storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/queries/RawQueryTest.java

```diff
@@ -65,6 +65,16 @@ public void observesTablesShouldNotBeNull() {
     }
 
     @Test
+    public void observesTagsShouldNotBeNull() {
+        RawQuery rawQuery = RawQuery.builder()
+                .query("lalala I know SQL")
```

Comment:
I won't tell @thevery and Stasic! (If you see this message it's a GitHub's bug obviously).